src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -119,8 +118,13 @@ def launch_pipeline(params):
                                '-o', 'UserKnownHostsFile=/dev/null',
                                '-o', 'StrictHostKeyChecking=no',
                                'screen', '-dmS', params.cluster_name])
-        # Run command on screen session
-        
+        # Run command on screen session  
+
+        if params.reference_genome == 'GRCh38':
+            from toil_scripts.adam_uberscript.input_files import GRCh38_inputs as inputs
+        if params.reference_genome == 'hg19':
+            from toil_scripts.adam_uberscript.input_files import hg19_inputs as inputs       
```

Comment:
My strong preference is to validate arguments as early as is possible; my understanding is that by setting the `choices` param when argparsing, we push the validation to when the arguments are parsed and cannot get invalid inputs here. I'd prefer to document this decision in the argparse section of the code, and to not revalidate here.

Also, this is personal preference, but I prefer to `assert` if an internal invariant fails, and to raise an exception (e.g., `IllegalArgumentException` in Java, `ValueError` in Python) if a user provides bad input. I find `assert False` to be a smell; if you're `assert False`ing, you should almost always `raise Exception("informative error message")` instead. In my world, if an `assert` fires, that indicates that there is a bug in the program that should be fixed, while raising an exception to the user tells them that they need to fix their inputs.